src/UI/chat/Chat.vue

```diff
@@ -73,14 +73,16 @@ export default {
       }
     },
     onEnter(event) {
-      this.checkMoveTop()
-      this.lastTexts.push({
-        text: event.target.value,
-        transformY: 0,
-        id: id++,
-      })
-      this.currentText = ''
-      this.$refs.chatBox.value = ''
+      if (event.target.value) {
+        this.checkMoveTop()
+        this.lastTexts.push({
+          text: event.target.value,
+          transformY: 0,
+          id: id++,
+        })
+        this.currentText = ''
+        this.$refs.chatBox.value = ''
+      }
```

Comment:
I personaly like to avoid nested levels by reversing logic. I would write it like this:
```js
const { value } = event.target

if (!value) {
  return
}

this.checkMoveTop()
this.lastTexts.push({
  text: value,
  transformY: 0,
  id: id++,
})

this.currentText = ''
this.$refs.chatBox.value = ''
```